tests/utils/is-caret-at-line-end.test.ts

```
@@ -0,0 +1,77 @@
+import { isCaretAtLineEnd } from '@/utils/is-caret-at-line-end';
```

ESLint

[error] Expected 2 empty lines after import statement not followed by another import.

(import/newline-after-import)